frontend/src/v4/routes/components/permissionsTable/permissionsTable.component.tsx

```diff
@@ -110,7 +112,7 @@ interface IProps {
 	context?: string;
 	onSelectionChange?: (selectedUsers) => void;
 	onFilterChange?: (filteredUsers) => void;
-	onPermissionsChange?: (permissions) => void;
+	onPermissionsChange?: (permissions, permissionsType, permissionsCount?) => void;
```

Comment:
dont change base components that are being used in other parts of the system (like in invitationdialogs)